spanner/unit_tests/test_client.py

```diff
@@ -106,6 +106,43 @@ def test_constructor_credentials_wo_create_scoped(self):
         expected_scopes = None
         self._constructor_test_helper(expected_scopes, creds)
 
+    def test_admin_api_lib_name(self):
+        """Establish that the lib_name and lib_version are passed to
+        the database and instance API objects in the GAPIC.
+        """
```

Comment:
:-1: on docstring for unit test (and on multi-line single-line docstring)

Can't recall how we resolved the discussion before